doc/changelog.rst

```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+1.0.0
+=====
+
+Backwards incompatible changes
+------------------------------
+
+* Python 2.6 is no longer supported
```

Comment:
I figured this way I'd be _forced_ to write the release note instead of putting it off until I want to tag the release :smile: